python/prairielearn.py

```diff
@@ -1515,21 +1520,21 @@ def latex_from_2darray(A, presentation_type="f", digits=2):
     Otherwise, each number is formatted as '{:.{digits}{presentation_type}}'.
     """
     # if A is a scalar
-    if np.isscalar(A):
```

Comment:
This is a minor mistake that appears in a bunch of places, see the notes on the docs for `isscalar`: https://numpy.org/doc/stable/reference/generated/numpy.isscalar.html